en/computer_vision/collision_prevention.md

```diff
@@ -2,78 +2,92 @@
 
 *Collision Prevention* may be used to automatically slow and stop a vehicle before it can crash into an obstacle.
 
-It can be enabled for multicopter vehicles in [Position mode](../flight_modes/position_mc.md), and can use sensor data from an offboard companion computer, a rangefinder attached to the flight controller, or both (fused).
+It can be enabled for multicopter vehicles in [Position mode](../flight_modes/position_mc.md), and can use sensor data from an offboard companion computer, offboard rangefinders over MAVLink, a rangefinder attached to the flight controller, or any combination of the above.
 
-> **Warning** Collision prevention may not prevent a crash if your vehicle is moving too fast!
-  This feature has only been tested (at time of writing) for a vehicle moving at 4 m/s.
+Collision prevention may restrict vehicle maximum speed if the sensor range isn't large enough!
+It also prevents motion in directions where no sensor data is available (i.e. if you have no rear-sensor data, you will not be able to fly backwards).
 
+> **Tip** If high flight speeds are critical, consider disabling collision prevention when not needed.
```

Comment:
FYI Note, I split the specific tip from the associated cause. Not 100% sure this is better, but I think so.